exercises/js-logic.js

```diff
@@ -36,9 +50,7 @@ Finish the function. Using the string that is passed in return the same string b
 
 Ex: If the string that is passed in is `hey there person` the return value should be `hey ereht nosrep`
 */
-const stringReverse = (str) => {
-	// answer here
-}
+const stringReverse = (str) => str.split(" ").map((word) => (word.length >= 5) ? word.split("").reverse().join("") : word).join(" ");
```

Comment:
I 100% would rather you gave it an attempt and be wrong than look up the answer on SOF and paste it in here